pack_info.cfg

```diff
@@ -0,0 +1,3 @@
+testpack/
```

Comment:
config file used by pad publishing

Reply:
Currently commenting for this file is not supported: https://github.com/amanj120/godot/blob/f176e948c353e45ea2d63dd2f200ca16d0f683b7/platform/android/export/gradle_export_util.h#L392

Reply:
Documentation for this configuration file created bay Aman can be found [here](https://github.com/amanj120/godot/blob/f176e948c353e45ea2d63dd2f200ca16d0f683b7/platform/android/export/export.cpp#L1597)